tests/python/test_build_data.py

```
@@ -2190,6 +2099,45 @@ def test_normalize_place_page_enrichment_rejects_ui_display_name_and_review_desc
+            "That’s gotta be one of the best hot chocolate drink I’ve tasted in my life! "
+            "The long wait was definitely worth it. There was quite a line before we got "
+            "a seat, but boy was it worth every minute. Definitely recommend this place."
+        )
+
+        enrichment = build_data.normalize_place_page_enrichment(details)
+
+        self.assertIsNone(enrichment.description)
+
+        for review_description in (
+            "It was my first attempt to eat mukhata. The food was so delicious and the clerks were very kind.",
+            "Best place to stay in Hanoi. I’d just finished Ha Giang loop and needed a place to rest.",
```

Ruff (0.15.12)

[warning] String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)